base_rest_demo/tests/common.py

```diff
@@ -29,14 +29,18 @@ def setUpClass(cls):
         cls.new_api_services_env = WorkContext(
             model_name="rest.service.registration", collection=collection
         )
-        cls.setUpExtendable()
+        cls.init_extendable_registry()
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+        cls.reset_extendable_registry()
```

Comment:
```suggestion
        cls.reset_extendable_registry()
        super().tearDownClass()
```

to tear down in reverse order as setup?

Reply:
done